packages/compass-telemetry/src/telemetry-events.ts

```diff
@@ -692,27 +692,27 @@ export type ExtraConnectionData = {
   /**
    * Specifies if KMS AWS is used.
```

Comment:
Should we update the docs here?

Reply:
added in [f3ac10f](https://github.com/mongodb-js/compass/pull/6235/commits/f3ac10f36e1df1e011a758030f24689ffabfe107)